rbc/irtools.py

```diff
@@ -287,6 +288,11 @@ def compile_instance(func, sig,
         flags.set('no_cpython_wrapper')
         flags.set('no_cfunc_wrapper')
 
+    # set CAPTURED_ERROS to "new_style"
+    # https://github.com/numba/numba/pull/7397
+    os.environ["CAPTURED_ERRORS"] = "new_style"
```

Comment:
Currently, the environment is set and the config is reloaded on all `compile_instance` calls.

I suggest setting the environment before importing numba, that is, in `rbc/__init__.py` have the following code:
```python
os.environ["CAPTURED_ERRORS"] = "new_style"
if 'numba' is sys.modules:
    numba.core.config.reload_config()
```
(untested, just illustrates the idea).